src/meta/src/hummock/manager/mod.rs

```diff
@@ -1479,6 +1468,21 @@ where
             }
         };
 
+        // 2. select_compactor
+        let compactor = self.compactor_manager.random_compactor(Some(&compact_task));
```

Comment:
`random_compactor` and `next_compactor` used for `test` and `trigger_manual_compaction`. IIUC, we can remove them and use `next_idle_compactor` of policy

Reply:
For vacuum or validation tasks, we might not need an idle compactor, because these tasks are more light-weight, and might not need to be skipped/canceled simply because there's no idle compactor.